src/components/Modal/ValidateProposalModal.tsx

```diff
@@ -229,7 +235,7 @@ function ValidateProposalModal({ proposal, account }: { proposal: IProposal; acc
                   onClick={() => onSubmit()}
                   type='button'
                   className='hover:text-green-600 hover:bg-green-50 bg-redpraha text-white rounded-xl px-5 py-2.5 text-center'>
-                  {isProposalUseEth ? 'Validate' : 'Allow spending'}
+                  {isProposalUseEth ? 'Validate awesome' : 'Allow spending'}
```

Comment:
haha don't forget to clean all debug and console log

Reply:
🤦 
I Need to be more careful, especially in PRs this large